docs/api-reference/instances/attributeChange.md

```diff
@@ -0,0 +1,51 @@
+<nav class="fusiondoc-api-breadcrumbs">
+	<a href="../..">Fusion</a>
+	<a href="..">Instances</a>
+</nav>
+
+<h1 class="fusiondoc-api-header" markdown>
+	<span class="fusiondoc-api-icon" markdown>:octicons-code-24:</span>
+	<span class="fusiondoc-api-name">AttributeChange</span>
+	<span class="fusiondoc-api-pills">
+		<span class="fusiondoc-api-pill-type">function</span>
+		<span class="fusiondoc-api-pill-since">since v0.3</span>
+	</span>
+</h1>
+
+Given an attribute name, returns a [special key](./specialkey.md) which connects
+to that attribute's change events.
+
+This function must be used with the [attribute](./attribute.md) special key otherwise the callback will never run.
```

Comment:
This is not necessarily true. There is nothing special about instances made with Fusion; GetAttributeChangedSignal can be invoked at any time, either by the developer via the Properties panel or by other scripts setting attributes on the instance.

Plus, this sentence implicitly advocates for a bad practice - we do not encourage people to use OnChange for properties they're also assigning a state object to, because that's unnecessary - if you have access to the state object, it's better to directly observe the state object than to go through Roblox to do the same thing.

As a result, this sentence should be removed.

Reply:
Oops! Didn't realise it was bad practice haha, will fix!